css/opengazettes.scss

```diff
@@ -0,0 +1,3 @@
+.jumbotron h1 {
+  color: fuchsia !important;
```

Comment:
@longhotsummer @smmbll I know !important is fugly.

This file's contents eventually ends up in _custom which is included before everything else. In the end this will get overridden by _home styling on the same selector. Is there a sass-y (had to) way to override with the includes in this order or should should _custom be included last? My early-css-era HTML brain thinks the latter.